pkg/arvo/sys/hoon.hoon

```diff
@@ -6310,6 +6269,72 @@
 ++  soft
   |*  han=$-(* *)
   |=(fud=* (mole |.((han fud))))
+::  +vi: virtualization engine
+::
+++  vi
+  ::  forward namespace?
+  ::
+  =/  for=?  &
+  ~%  %vi  +>
+    ==
+      %mure  mure
+      %mute  mute
+    ==
+  |%
```

Comment:
Style nit: this can just be `|_  for=?`, and the parent core can be specified by wing: `~%  %vi  ..vi`.